HISTORY.rst

```diff
@@ -7,6 +7,10 @@ Pending Release
 
 * (Insert new release notes below this line)
 
+1.0.1 (2017-04-27)
```

Comment:
don't insert the version number header, that's done in a release commit like 0642f4454de10d61c067f0c8ff7092de475ee7ab